.kres.yaml

```diff
@@ -4,7 +4,6 @@ spec:
   targets:
     # - non-related to the kernel, in alphabetical order
     - apparmor
-    - base
```

Comment:
It's used by extensions as the tools base package. However we might probably want to use tools directly there in place of pkgs/base

Reply:
yeh my intention was to use tools directly there

Reply:
Should be good